src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java

```diff
@@ -0,0 +1,101 @@
+package mate.jdbc.dao;
+
+import mate.jdbc.lib.Dao;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.util.ConnectionUtil;
+import mate.jdbc.util.DataProcessingException;
+
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Dao
+public class ManufacturerDaoImpl implements ManufacturerDao {
+    @Override
+    public List<Manufacturer> getAll() {
+        List<Manufacturer> getAllManufacturers = new ArrayList<>();
+        try (Connection connection = ConnectionUtil.getConnection();
+             Statement getAllInitsStatement = connection.createStatement()) {
```

Comment:
@EliosPride getAllInitsStatement -> statement (rename in all methods)

Reply:
@Garret-vin ++